src/libecalc/fixtures/cases/consumer_system_v2/data/consumer_system_v2.yaml

```diff
@@ -102,19 +102,18 @@ INSTALLATIONS:
               COMPRESSOR_MODEL: compressor_sampled_1d
             - NAME: compressor3
               COMPRESSOR_MODEL: compressor_sampled_1d
+          COMPONENT_CONDITIONS:
```

Comment:
COMPONENT_PROPERTIES? Or use more specific naming? I'm thinking this can contain CONDITION (rename to ENABLED or something?), if we add a start and end we could have that here. TOTAL_SYSTEM_RATE could be here for system, but it (still) seems unnecessary to me.